packages/logs/lib/models/messages.ts

```diff
@@ -0,0 +1,129 @@
+import { client } from '../es/client.js';
+import type { MessageRow } from '../types/messages.js';
+import { indexMessages } from '../es/schema.js';
+
+export interface ListOperations {
+    count: number;
+    items: MessageRow[];
+}
+export interface ListMessages {
+    count: number;
+    items: MessageRow[];
+}
+
+/**
+ * Create one message
+ */
+export async function createMessage(row: MessageRow): Promise<void> {
+    await client.create<MessageRow>({
+        index: indexMessages.index,
+        id: row.id,
+        document: row,
+        refresh: true
+    });
+}
+
+/**
+ * List operations
+ */
+export async function listOperations(opts: { limit: number }): Promise<ListOperations> {
+    const res = await client.search<MessageRow>({
+        index: indexMessages.index,
+        size: opts.limit,
+        sort: [{ createdAt: 'desc' }, '_score'],
+        track_total_hits: true,
+        query: {
+            // @ts-expect-error I don't get the error
```

Comment:
what's the error?

Reply:
The query is valid and it works but the typings is not happy, could not find any way to make it pass